### DISCUSSION TITLE
Are there plans for a Vim mode?

### DISCUSSION THREAD

**Discussion by ReedMerrill** [marimo-team/marimo#593] [Q&A] [answered]
This looks awesome, but for people who use vim mode in their current editor it's tough to switch to a browser based editor that likely doesn't support vim-style editing. This is also one of my biggest pain points with Jupyter Notebooks, and It's the reason I edit them in VS Code with a Vim mode extension. 

Are there plans to implement a Vim mode for this project?

**Comment by roberthobblebottom** [comment, marked as answer]
In the interface, click on settings and go down to keymap section, there is a vim keymap option to use vim controls.  
  
`jupyter-lab` has plugin for vim too. not sure about jupyter-notebook.
